<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item alignment="Qt::AlignHCenter">
            <widget class="QProgressBar" name="progressListA">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>133</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>133</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QLabel" name="labelListA">
             <property name="maximumSize">
              <size>
               <width>43</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>Listy
 x/10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item alignment="Qt::AlignHCenter">
            <widget class="QProgressBar" name="progressMalaA">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>133</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>133</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="labelMalaA">
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>52</height>
              </size>
             </property>
             <property name="text">
              <string>Paczki
 Małe
 x/10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item alignment="Qt::AlignHCenter">
            <widget class="QProgressBar" name="progressSredniaA">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>137</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="labelSredniaA">
             <property name="maximumSize">
              <size>
               <width>39</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>Paczki
Średnie
 x/10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item alignment="Qt::AlignHCenter">
            <widget class="QProgressBar" name="progressDuzaA">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>137</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="labelDuzaA">
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>Paczki
Duże
 x/10</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelMagazynuA">
         <property name="text">
          <string>Magazyn A</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item alignment="Qt::AlignHCenter">
            <widget class="QProgressBar" name="progressListB">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>133</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="labelListB">
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>52</height>
              </size>
             </property>
             <property name="text">
              <string>Listy
 x/10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item alignment="Qt::AlignHCenter">
            <widget class="QProgressBar" name="progressMalaB">
             <property name="minimumSize">
              <size>
               <width>24</width>
               <height>133</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>133</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="labelMalaB">
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>52</height>
              </size>
             </property>
             <property name="text">
              <string>Paczki
 Małe
 x/10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item alignment="Qt::AlignHCenter">
            <widget class="QProgressBar" name="progressSredniaB">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>137</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="labelSredniaB">
             <property name="maximumSize">
              <size>
               <width>39</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>Paczki
Średnie
 x/10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item alignment="Qt::AlignHCenter">
            <widget class="QProgressBar" name="progressDuzaB">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>137</height>
              </size>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="labelDuzaB">
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>48</height>
              </size>
             </property>
             <property name="text">
              <string>Paczki
Duże
 x/10</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelMagazynuB">
         <property name="text">
          <string>Magazyn B</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>84</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <widget class="QPushButton" name="usunOstatniaButton">
       <property name="text">
        <string>Usuń Ostatnią</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="oproznijMagazynButton">
       <property name="text">
        <string>Opróżnij
magazyn</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>258</width>
      <height>250</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="zLabel">
           <property name="text">
            <string>Z</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboZ">
           <item>
            <property name="text">
             <string>Mag A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mag B</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="przeniesButton">
         <property name="text">
          <string>Przenieś</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <widget class="QLabel" name="doLabel">
           <property name="text">
            <string>Do</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboDo">
           <item>
            <property name="text">
             <string>Mag A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mag B</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="historiaPaczek"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>203</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_16">
     <item>
      <widget class="QComboBox" name="magazynComboBox">
       <property name="maximumSize">
        <size>
         <width>201</width>
         <height>24</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Magazyn A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magazyn B</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="infoText">
         <property name="maximumSize">
          <size>
           <width>199</width>
           <height>70</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="losujButton">
         <property name="maximumSize">
          <size>
           <width>199</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Losuj Przesyłkę</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>losujButton</tabstop>
  <tabstop>historiaPaczek</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
